index.html

```diff
@@ -270,87 +291,285 @@
 It will continue to evolve and the task force will issue updates as often as needed. At the
 conclusion of the task force, the TAG intends to adopt this document as a Finding.
 
+<aside class="note">
+
+Privacy is covered by legal frameworks and this document recognises that existing data
+protection laws take precedence for legal matters. However, because the Web is global, we
+benefit from having shared concepts to guide its evolution as a system built for the people using it
+([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
+can hopefully help all the Web's participants in different legal regimes. Our shared
+understanding is that the law is a floor, not a ceiling.
```

Comment:
Right — I've gone back and forth on this one a lot. There's nowhere in the text that it flows well. The idea is that it should be said early because that point needs to be addressed. It might not be SotD but it's definitely front matter material IMHO, and I don't think that we can just make up front matter sections (and what would we call it?).

So I've landed on SotD being the least bad option. I'm pretty sure that there's precedent for this kind of decision.